sdk/keyvault/Azure.Security.KeyVault.Certificates/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting Azure Key Vault Certificates SDK Issues
+
+The `Azure.Security.KeyVault.Certificates` package provides APIs for operations on Azure Key Vault for the
+`CertificateClient` class. This troubleshooting guide contains mitigation steps for issues specific to the
```

Comment:
```suggestion
`CertificateClient` class. This troubleshooting guide contains steps for diagnosing issues specific to the
```

Reply:
Same in a few other places

Reply:
Did @schaabs change this in his upcoming PR? I copied verbatim what he used.

Reply:
I don't know - I just think diagnosing makes more sense, since we may not always have a definite solution, but the guide may help in narrowing it down at least